tests/Services/GamServiceTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Tests\Services;
+
+use LiveIntent\Exceptions\InvalidRequestException;
+
+class GamServiceTest extends ServiceTestCase
+{
+    protected $serviceKey = 'gam';
+
+    public function testCreateGamUrl()
+    {
+        $adSlotId = 834557;
```

Comment:
Can we use a fixture id instead? As in, something that we know to already exist there?

Reply:
Sure